src/components/popover/popover.stories.ts

```diff
@@ -101,3 +101,29 @@ export const flipPlacements_TestOnly = (): string => html`
     document.querySelector(".my-popover").flipPlacements = ["right"];
   </script>
 `;
+
+export const scaleConsistencyPopoverHeadingActionSlottedIcon = stepStory(
+  (): string => html`
+    <div style="width: 400px;">
+      ${referenceElementHTML}
+      <calcite-popover
+        heading="Dreams didn't make us kings. Dragons did. 🐉"
+        reference-element="reference-element"
+        placement="${select("placement", placements, defaultPopoverPlacement)}"
+        ${boolean("open", true)}
+        ${boolean("dismissible", true)}
+        scale="${select("scale", ["s", "m", "l"], "m")}"
+      >
+        ${contentHTML}
+      </calcite-popover>
+    </div>
+  `,
+  createSteps("calcite-popover")
```

Comment:
Let's drop the stepped stories to keep our screenshot quota down.

Reply:
Ok, I added them for designers to be able to give me feedback on visuals. I'll prep a demo instead. I think having an interactive demo would be better for a design review anyway.